examples/Images and Tables.ipynb

```diff
@@ -219,7 +231,36 @@
    },
    "outputs": [],
    "source": [
-    "tv"
+    "from firefly_widgets import ChartViewer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "cv = ChartViewer(layout=Layout(width='600px', height='400px'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cv.keys"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cv"
```

Comment:
I am unable to add a line comment for the next cell since it isn't a change. When I execute the next cell with `tv.filters = 'w3snr > 10'`, the table widget displays some kind of SQL error message, and the chart widget goes blank. Does this happen for you?

In the widget, the message is
```
Failed to load table. Error: BadSqlGrammarException:StatementCallback; bad SQL grammar [CREATE TABLE data_95c2e07c05697cab9f6ce9e4d39bdd8e AS (select b.*, (ROWNUM-1) as ROW_NUM from (select "designation", "ra", "dec", "clon", "clat", "sigra", "sigdec", "sigradec", "w1mpro", "w1sigmpro", "w1snr", "w1rchi2", "w2mpro", "w2sigmpro", "w2snr", "w2rchi2", "w3mpro", "w3sigmpro", "w3snr", "w3rchi2", "w4mpro", "w4sigmpro", "w4snr", "w4rchi2", "nb", "na", "w1sat", "w2sat", "w3sat", "w4sat", "pmra", "sigpmra", "pmdec", "sigpmdec", "cc_flags", "ext_flg", "var_flg", "ph_qual", "moon_lev", "w1nm", "w1m", "w2nm", "w2m", "w3nm", "w3m", "w4nm", "w4m", "dist", "angle", ROW_IDX from data where (w3snr > 20) ) as b) WITH DATA]; nested exception is java.sql.SQLSyntaxErrorException: user lacks privilege or object not found: W3SNR from:null
```

If this is not easily resolved, I suggest to comment out the filters cell...or since we are still calling this '0.1.0b6' we could leave it since the version indicates this is unstable early development.